_posts/2021-01-29-status-update.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "2021-01-29: IAC Team Status Update"
+author: tphoney
+categories:
+  - team
+  - status
+tags:
+  - litmus
+  - cloud-ci
+  - modules
+  - iac-team
+  - community-day
+---
+
+Hey everyone, it's my honour to write my first blogpost for Puppet. Kinda strange in [I][TP] have done a number of other things for puppet, but never a blogpost. Case in point, my talk about [litmus](https://www.youtube.com/watch?v=FYfR7ZEGHoE) (available permanently on youtube to make me shudder at my own voice). On to business !!!
+
+## What is the IAC Team
+
+How did you find this post then ? Just kidding, we are a team at puppet that looks after all the modules / tasks / gems / tools that you as a community use. If we dont look after it we know who you should talk to. Here is a great blog post on how to reach us [Puppet IAC Team](https://puppetlabs.github.io/iac/team/2021/01/20/reaching-out.html)
+
+## Ecosystem Kickoff
+
+The IAC team is part of a larger group at puppet that has as its main focus 'you' the community. We had our opening meeting discussing the direction of our work for the next year, and it all looks very exciting. I can't give away too much yet, but more details will trickle out as we confirm our goals.
+
+## Automated Module Release
+
+What I can talk a little about is [Danny's][Danny] work on a new proof of concept. We are an automation company so we are always looking to remove a few steps from our process, this project will enable us to automate some of the manual steps required to release modules. Leaving the the human to only review and click merge on Github.
```

Comment:
@carabasdaniel , do you have a link to the work handy, so that folks can have a peek on what's cooking?